masonite/commands/ServeCommand.py

```diff
@@ -9,14 +16,276 @@ class ServeCommand(Command):
     serve
         {--port=8000 : Specify which port to run the server}
         {--host=127.0.0.1 : Specify which ip address to run the server}
+        {--reload : Make the server automatically reload on file changes}
```

Comment:
This is all awesome. The only change I can think of change the `--reload` to `--r|--reload` This will allow us to run `craft serve -r` as well as `craft serve --reload`

Reply:
see the ControllerCommand for an example on how cleo handles multiple options